src/screens/updates/UpdatesScreen.tsx

```diff
@@ -84,7 +84,7 @@ const UpdatesScreen = ({ navigation }: UpdateScreenProps) => {
           e.preventDefault();
 
           navigation.navigate('MoreStack', {
-            screen: 'Downloads',
```

Comment:
on mihon this sends u to the page that shows the chapters currently being downloaded, so task queue is a closer match